src/test/java/pl/stormit/eduquiz/game/controller/GameControllerTest.java

```diff
@@ -78,6 +107,28 @@ void shouldReturn200WhenFoundGameByIdCorrectly() throws Exception {
         result.andExpect(status().isOk());
     }
 
+    @Test
+    void shouldReturn404WhenFoundGameByIdInCorrectly() throws Exception {
```

Comment:
Are you sure application should return 404 status if game found by ID correctly?

Reply:
There is extra capital letter. I mean Incorrectly instead of InCorrectly. I will change the method name to more readable.